tests/test_cli.py

```diff
@@ -108,3 +106,13 @@ def test_sum(capsys):
 
     out, err = capsys.readouterr()
     assert out.strip() == tag.pkgdata('sum-test-out.txt').read().strip()
+
+
+def test_merge(capfd):
```

Comment:
There's some capsys/capfd weirdness that needs to be sorted out. May as well drop the whole logging crap as it seems to create a lot of problems with capturing testing output.